sonic_package_manager/service_creator/creator.py

```diff
@@ -553,12 +553,17 @@ def update_config_with_init_cfg(cfg, conn):
                     key_cfg.update(fvs)
                     conn.mod_entry(table, key, key_cfg)
 
-        for conn in self.sonic_db.get_connectors():
-            cfg = conn.get_config()
-            new_cfg = init_cfg.copy()
-            utils.deep_update(new_cfg, cfg)
-            self.validate_config(new_cfg)
-            update_config_with_init_cfg(cfg, conn)
+        # Get running database connector directly
+        conn = self.sonic_db.get_running_db_connector()
+        if conn is None:
+            log.error("No running database available")
+            return
+
+        cfg = conn.get_config()
+        new_cfg = init_cfg.copy()
+        utils.deep_update(new_cfg, cfg)
+        self.validate_config(new_cfg)
+        update_config_with_init_cfg(cfg, conn)
```

Comment:
If we build image wih extension - how the default config_db.json will get initial config ?